content/en/quickstart/_index.md

```diff
@@ -46,11 +46,13 @@ This will install the Istio demo profile with ["Istio core" "Istiod" "Ingress ga
 ✔ Installation complete
 ```
 
-After the previous step is completed - the validation can be done by confirming GetMesh and Istio version installed:
+<h3>Verify it's Worked!</h3>
```

Comment:
typo: "it worked"

Reply:
also, use "###" for h3

Reply:
Oddly, the original page used HTML markup rather than markdown, so I preserved that in my edits.  Will convert all to markdown.